src/pykeen/constants.py

```diff
@@ -44,3 +47,18 @@
 USER_DEFINED_CODE = "<user defined>"
 
 AGGREGATIONS = {func.__name__: func for func in [torch.sum, torch.max, torch.mean, torch.logsumexp]}
+
+# constants for sides
+COLUMN_HEAD = 0
+COLUMN_RELATION = 1
+COLUMN_TAIL = 2
+LABEL_HEAD: Side = "head"
+LABEL_RELATION: Side = "relation"
+LABEL_TAIL: Side = "tail"
+# TODO: extend to relation, cf. https://github.com/pykeen/pykeen/pull/728
+SIDES: Tuple[Side, ...] = (LABEL_HEAD, LABEL_TAIL)
+PART_TO_COLUMN = {
+    LABEL_HEAD: COLUMN_HEAD,
+    LABEL_RELATION: COLUMN_RELATION,
+    LABEL_TAIL: COLUMN_TAIL,
+}
```

Comment:
@cthoyt maybe we want to have an extra (earlier) PR for this.

Reply:
I'd support doing a PR for just moving of constants/update of type annotations